docs/connector-development/testing-connectors/source-acceptance-tests-reference.md

```diff
@@ -121,6 +121,8 @@ Verifies when a discover operation is run on the connector using the given confi
 | `config_path` | string | `secrets/config.json` | Path to a JSON object representing a valid connector configuration |
 | `configured_catalog_path` | string | `integration_tests/configured_catalog.json` | Path to configured catalog |
 | `timeout_seconds` | int | 30 | Test execution timeout in seconds |
+| `backward_compatibility_tests_config.previous_connector_version` | string | `latest` | Previous connector version to use for backward compatibility tests (expects a version following semantic versioning). |
```

Comment:
thinking out loud: is there a way to autogenerate this doc from our pydantic models instead of manually doing this? 🤔

Reply:
[Sphinx-Pydantic](https://sphinx-pydantic.readthedocs.io/en/latest/): But this would mean migrating our SAT reference to Sphinx 

[JSON Schema for humans](https://coveooss.github.io/json-schema-for-humans/#/): can render markdown, but we need to serialize pydantic models to JSON schema